onnxruntime/core/framework/execution_provider.cc

```diff
@@ -131,6 +132,11 @@ void ExecutionProviders::LogProviderOptions(const std::string& provider_id,
         TraceLoggingString(config_pair.second.c_str(), "Value"),
         TraceLoggingBool(captureState, "isCaptureState"));
   }
+#else
+      ORT_UNUSED_PARAMETER(provider_id);
+      ORT_UNUSED_PARAMETER(providerOptions);
+      ORT_UNUSED_PARAMETER(captureState);
```

Comment:
linter is complaining about the indent here

Reply:
Done